src/olympia/constants/activity.py

```diff
@@ -1,7 +1,3 @@
-# The absolute import feature is required so that we get the root celery
-# module rather than `amo.celery`.
```

Comment:
After reading the original commit I still have no idea what the `absolute_import` in this file had to do with celery as it doesn't reference celery at all and these constants aren't related to it either.

Reply:
It made sense when it imported celery a while ago but that was removed in 133827601c02a5f6c3aca3c1db3c86f40f2ca685